crates/biome_html_formatter/tests/specs/html/elements/bracket-same-line/self-closing.html

```diff
@@ -0,0 +1,7 @@
+<selfclosing
+  id="hello"
+  class="world"
+  style="color: red"
+  data-foo="bar"
+  data-bar="foo"
+/>
```

Comment:
If we need to follow the spec, self closing elements don't exist in HTML. Perhaps we can add another case

Reply:
Self closing elements technically do exist, they are called "void elements" in the spec, and they are defined as elements that have no children, and therefore do not have a closing tag. Syntactically, they do not need the `/` and it is ignored by browsers. Prettier always adds the `/` for void elements.

Reply:
That's something we might want to discuss later, but I was thinking of diverging from Prettier and not adding the slash at the end. Maybe we could consider an option